src/components/v2/appDetails/k8Resource/nodeDetail/nodeDetail.api.ts

```diff
@@ -410,14 +367,15 @@ export const deleteEphemeralUrl = (
     appName: string,
```

Comment:
Can we transform params into object for deleteEphemeralUrl and generateEphemeralUrl